src/mbgl/style/layers/layer_properties.hpp.ejs

```diff
@@ -37,7 +37,13 @@ public:
 <% if (/-pattern$/.test(property.name) || property.name === 'line-dasharray') { -%>
     PaintProperty<<%- propertyType(property) %>, CrossFadedPropertyEvaluator> <%- camelizeWithLeadingLowercase(property.name) %> { <%- defaultValue(property) %> };
 <% } else if (property.name === 'fill-outline-color') { -%>
-    PaintProperty<<%- propertyType(property) %>> <%- camelizeWithLeadingLowercase(property.name) %> { { 0, 0, 0, -1 } };
+    PaintProperty<<%- propertyType(property) %>> <%- camelizeWithLeadingLowercase(property.name) %> { {} };
```

Comment:
I tried this before, but the current implementation of `PaintProperty` does not allow an empty constructor like `PropertyValue` does:
```
../../src/mbgl/style/layers/fill_layer_properties.hpp:23:43: error: no matching constructor for initialization of 'PaintProperty<mbgl::Color>'
    PaintProperty<Color> fillOutlineColor {};
                                          ^~
```
Are you proposing then to have `PaintProperty` use the same variant mechanism to store its default value?